src/lib/base.ts

```diff
@@ -29,3 +31,16 @@ export interface PostgresConnector {
   topics: {table_name: string; topic_name: string}[];
   uuid: string;
 }
+
+export function confirmConnector(connector: string, confirm: string | undefined) {
```

Comment:
This could be `export async function` to avoid having to deal with the Promise plumbing. That will pair nicely with  a try/catch in `destroy`; we could need to use `cli.error` to raise an error if there is a mismatch.